shared-modules/README.md

```diff
@@ -0,0 +1,29 @@
+This repository contains commonly shared modules and is intended to be used as a git submodule.
```

Comment:
Looks like you copied the files over instead of using the shared-modules as a submodule with git.